src/app/app.component.html

```diff
@@ -2,19 +2,16 @@
     <jsdaddy-open-source-header [githubMaskLink]="githubMaskLink"></jsdaddy-open-source-header>
     <section class="mask-wrap row">
         <div class="drawer-container column">
-            <jsdaddy-open-source-accordion
-                [lists]="lists"
-                (itemAccordion)="switchDoc($event)"
-                (itemInAccordion)="chosenItemS($event)">
+            <jsdaddy-open-source-accordion [lists]="lists" (itemAccordion)="switchDoc($event)">
```

Comment:
use selector which does not have content without closing tag
For example `<jsdaddy-open-source-accordion> -> </jsdaddy-open-source-accordion>` -> `<jsdaddy-open-source-accordion />`